activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb

```diff
@@ -111,13 +111,15 @@ def #{sym}=(obj)
   #   Account.user     # => "DHH"
   #   Account.new.user # => "DHH"
   #
+  # Unlike `mattr_accessor`, values are *not* shared with subclasses or parent classes.
   # If a subclass changes the value, the parent class' value is not changed.
-  # Similarly, if the parent class changes the value, the value of subclasses
-  # is not changed.
+  # If the parent class changes the value, the value of subclasses is not changed.
   #
   #   class Customer < Account
   #   end
   #
+  #   Account.user  # => "DHH"
+  #   Current.user  # => nil
```

Comment:
Should this be Customer?
```suggestion
  #   Customer.user  # => nil
```